Dockerfile

```diff
@@ -6,18 +6,38 @@ RUN sudo apt-get update -y &&\
     sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 2 &&\
     sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 3 
 RUN python3 -m pip install --upgrade pip
+RUN echo "export ROS_PACKAGE_PATH=/src/aerialist/aerialist/inter_images:\$ROS_PACKAGE_PATH" >> /root/.bashrc
+
+RUN sudo apt install ros-melodic-stereo-image-proc ros-melodic-image-view -y
+# RUN sudo apt-get install python-rospy
 #&&\
 # sudo -H pip3 install --upgrade pip
 
 #Setting up the current tool 
 COPY ./requirements.txt /src/aerialist/requirements.txt
+WORKDIR /src/catkin_ws/src/avoidance/
```

Comment:
let's see if we still need this parts after possible changes to the obstacle spawn code